scripts/release.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 set -xe
 
+GOLANG="${GOLANG:-golang:1.21-alpine}"
```

Comment:
wondering where is `build.sh` in this script and found #961 :) happy to revert the changes to this file and rebase if that PR is merged.

Reply:
I'll merge that :-)